src/app/admin/page.tsx

```diff
@@ -78,12 +84,9 @@ function Admin() {
         )
     }
     return (
-        // <div>
-
-        // </div>
-        <Card loading={loading} title={user.company}>
+        <Card loading={loading} title={user.company_name}>
             <Space direction="vertical">
-                {user.role == "admin" ? <ProviderDetails user={user} /> : null}
+                {user != undefined ? <ProviderDetails user={user} /> : null}
```

Comment:
@sweiyang , provider API do not have role.

<img width="914" alt="image" src="https://github.com/NUS-EVCHARGE/ev-frontend-service/assets/10574652/f616de6e-02a3-4c97-9f6b-ee61eb39a7e3">